build/common.ps1

```diff
@@ -157,95 +157,81 @@ Function Install-DotnetCLI {
     param(
         [switch]$Force
     )
-    $vsMajorVersion = Get-VSMajorVersion
-    $MSBuildExe = Get-MSBuildExe $vsMajorVersion
-    $CliBranchListForTesting = & $msbuildExe $NuGetClientRoot\build\config.props /v:m /nologo /t:GetCliBranchForTesting
-    $CliBranchList = $CliBranchListForTesting.Split(';');
+    $env:DOTNET_HOME = $CLIRoot
```

Comment:
Can you elaborate what's the issue with the scripts that you are trying to fix?

Reply:
I mentioned the issues in the commit's description.

- MSBuild picks up response files automatically, which messes up reading from standard output.
- The overridden version won't run the tests successfully. Since, some tests error out by saying 'Pkcs' dll missing.
- Cleanup the logic to make it better understandable.

Reply:
> Since, some tests error out by saying 'Pkcs' dll missing.

That shouldn't be happening.

We test both 3.x and 5.x SDKs, that's why we download both

Reply:
I'll rebase over #3552 that seems to remove the workaround.